### DISCUSSION TITLE
Welcome to AGAT Discussions!

### DISCUSSION THREAD

**Discussion by Juke34** [NBISweden/AGAT#293] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by AsaArmDe** [comment]
What database AGAT is using for the prediction of proteins?

**Reply by Juke34** [reply]
AGAT does not perform any protein predictions.

**Reply by AsaArmDe** [reply]
Maybe question was not addressed well...In the output file, I have protein matches, single exon matches, ... how does it understand and count them?

**Reply by Juke34** [reply]
What AGAT script output are you talking about?

**Reply by AsaArmDe** [reply]
agat_sp_statistics.pl
